dist/components/list/_list.scss

```diff
@@ -0,0 +1,62 @@
+// List
+// ===
+//
+// Style list as a list
+
+
+// Component Variables
+// ---
+
+$list__margin: if(variable_exists(v-space), $v-space 0, 10px 0) !default;
+$list__padding-left: 15px !default;
+$list__item-margin: if(variable_exists(v-space), $v-space/2 0, 5px 0) !default;
+
+
+// List
+// ---
+//
+// 1. Keep bullets on the same vertical line as content
+
+.c-list {
+    margin: $list__margin;
```

Comment:
Components shouldn’t control their margins. That should be the responsibility of the layout that contains them.